frontend/testing/cypress/src/integration/studio/overview.js

```diff
@@ -5,8 +5,11 @@ import * as texts from '../../../../../language/src/nb.json';
 
 const designerAppId = `${Cypress.env('autoTestUser')}/${Cypress.env('designerAppName')}`;
 const orgAppId = `${Cypress.env('orgUserName')}/${Cypress.env('designerAppName')}`;
-const NEW_ADM_FEATURE_FLAG = 'newAdministration';
-const PROCESS_EDITOR_FEATURE_FLAG = 'processEditor';
+
+const FeatureFlagEnum = Object.freeze({
```

Comment:
JavaScript does not support enums as TypeScript does. But I have created a read-only object by using the `Object.freeze` which ensures the values cannot be changed runtime. The benefit of this is that we do not need to remember all the constants. We can type "FeatureFlagEnum." and IntelliSense within the IDE will help us out. 😎

Reply:
Excellent!